css/style.css

```diff
@@ -0,0 +1,1248 @@
+    width: auto;
+  }
+}
+
+
+
+/*
+PORTFOLIO SECTION
+*/
+
+.portfolio-container {
+  display: -webkit-flex;
+  display: flex;
+  -webkit-flex-direction: column;
+  flex-direction: column;
+  -webkit-align-items: center;
+  align-items: center;
+  max-width: 68em;
+  padding: 3.5em 0 0;
+  margin: 0 auto;
+}
+
+.portfolio-section-header {
+  width: 100%;
+  text-align: center;
+  margin-top: .2em;
+  margin-bottom: .5em;
+}
+
+@media screen and (min-width: 50em) {
+
+  .portfolio-section-header {
+    margin: 0 6%;
+    padding: 1em 0;
+    background-size: calc(100% - 11.5em);
+  }
+
+  .portfolio-container {
+    position: relative;
+    -webkit-flex-direction: row;
+    flex-direction: row;
+    -webkit-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -webkit-align-items: center;
+    align-items: center;
+    -webkit-justify-content: space-around;
+    justify-content: space-around;
+    padding-top: 2em;
+  }
+}
+
+
+
+/*
+PORTFOLIO SLIDER
+*/
+
+.portfolio-slider {
+  width: 100%;
+}
+
+.portfolio-slider-figure {
+  position: relative;
+  overflow: hidden;
+  direction: rtl;
+}
+
+.portfolio-slider-figure img {
+  min-width: 40em;
+  margin: 0 auto;
+}
+
+.portfolio-slider-overlay {
+  opacity: .7;
+}
+
+.portfolio-slider-caption {
+  position: absolute;
+  top: 10%;
+  right: 18%;
+  width: 14em;
+  line-height: 1.1;
+  text-align: left;
+  text-shadow: 1px 1px 5px #2e2e2e;
+}
+
+.portfolio-slider-caption p {
+  font-size: 1.333em;
+}
+
+.portfolio-slider-button {
+  padding: 1em 1em;
+  margin: .8em 0;
+  background-image: linear-gradient(#4dbff7, #5087c8);
+}
+
+.portfolio-slider-button:hover {
+  cursor: pointer;
+}
+
+.portfolio-slider-button:before,
+.portfolio-slider-button:after {
+  font-family: "ico-fonts";
+}
+
+
+.portfolio-slider-button:after {
+  content: "\e60a";
+  padding-left: 1.333em;
+}
+
+.portfolio-slider-button:before {
+  content: "\e610";
+  padding-right: 1em;
+  color: rgba(0,0,0,.3);
+}
+
+.portfolio-slider-bullets[aria-hidden="true"] {
+  visibility: hidden;
+}
+
+@media screen and (min-width: 50em) {
+
+  .portfolio-slider-caption {
+    top: 20%;
+    right: 25%;
+  }
+
+  .portfolio-slider-bullets[aria-hidden="true"] {
+    visibility: visible;
+  }
+
+  .portfolio-slider-bullets {
+    padding: .3em;
+    font-size: 1.777em;
+    text-align: center;
+  }
+
+  .portfolio-slider-bullets li {
+    display: inline;
+  }
+
+  [title$="bullet"]:before {
+    content: "•";
+    color: #d4d4d4;
+  }
+
+  .slider-bullet-active:before {
+    color: #fd6632;
+  }
+}
+
+
+
+/*
+PORTFOLIO NAVIGATION
+*/
+
+[title="portfolio-categories"] li {
+  display: block;
+  margin-top: .9em;
+  margin-bottom: .5em;
+  text-align: center;
+}
+
+.portfolio-menu-item {
+  padding: .3em;
+  font-size: 1.333rem;
+  text-decoration: none;
+  color: #d4d4d4;
+}
+
+.portfolio-selected-category {
+  color: #fff;
+  background-color: #fd6632;
+}
+
+@media screen and (min-width: 50em) {
+
+
+  [title="portfolio navigation"] {
```

Comment:
Rather than use the `title` attribute on this element, you should use a class (`.portfolio-navigation` works just fine). `title` is really meant more for adding accessibility, and _should_ be changeable without breaking your styles.